frontend/components/text-classifier/labeling-rules/RuleDefinition.vue

```diff
@@ -72,12 +95,16 @@ export default {
       await this.dataset.viewSettings.enableRulesSummary();
     },
     async updateRule() {
-      this.refresh++;
       await this.$fetch();
     },
+    updateLabel(label) {
+      this.selectedLabels = label;
```

Comment:
changing the selected label should change the rule metrics?

Reply:
yes, enables the communication of the current label to the metrics component